StabilityPackage.wl

```diff
@@ -678,3 +680,39 @@ Export[name<>".h5",{
 
 End[]
 EndPackage[]
+

```

Comment:
Is this supposed to be after EndPackage?

Reply:
Yes.